src/class-manifest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * The Manifest data specific functionality.
+ *
+ * @since   4.0.0 Init class.
+ * @package Inf_Theme\Core
+ */
+
+namespace Inf_Theme\Core;
+
+use Eightshift_Libs\Assets\Manifest as LibManifest;
+use Eightshift_Libs\Core\Service;
+
+/**
+ * Class Mainfest
+ */
+class Manifest extends LibManifest implements Service {
```

Comment:
You need to implement `Service` here? I thought this was handled in the lib? 🤔 Can you test and see if something fails? 🙂

Reply:
Lib only checks if the manifest.json exists and processes the file and returns it as a method.

In your project (boilerplate) you should assign that processed data to the global variable and use it as you like.

I think this gives us the best flexibility for the future